src/main/scala/mesosphere/marathon/core/group/impl/AssignDynamicServiceLogic.scala

```diff
@@ -79,16 +79,15 @@ object AssignDynamicServiceLogic extends StrictLogging {
      * app in the case that servicePorts are re-posted all as 0's.
      */
     val usedServicePorts: Set[Int] =
-      (from.transitiveApps.iterator ++ to.transitiveApps.iterator).flatMap(_.servicePorts).toSet
+      (from.transitiveApps ++ to.transitiveApps).flatMap(_.servicePorts).toSet
```

Comment:
We do not have to use an iterator here since concat of two vectors is fast enough.

Reply:
👏